tests/e2e/grpc-based-registration.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+import grpc
+import uuid
+
+import feast.core.CoreService_pb2
+from feast import Feature
+from feast.core.CoreService_pb2_grpc import CoreServiceStub
+from feast.feature_set import FeatureSet
+from feast import ValueType
+
+PROJECT_NAME = 'batch_' + uuid.uuid4().hex.upper()[0:6]
+LAST_VERSION = 0
+GRPC_CONNECTION_TIMEOUT = 3
+LABEL_KEY = "my"
+LABEL_VALUE = "label"
+
+
```

Comment:
The tests look good, but possibly add a `TODO` to migrate this over to the python sdk once that has been updated?